addons/spreadsheet/static/tests/lists/list_plugin.test.js

```diff
@@ -417,6 +497,84 @@ test("can edit list domain", async () => {
     expect(getCellValue(model, "B2")).toBe("");
 });
 
+test("can edit list sorting", async () => {
+    const { model } = await createSpreadsheetWithList({
+        columns: ["foo", "bar", "date", "probability", "pognon"],
+    });
+    const [listId] = model.getters.getListIds();
+    expect(model.getters.getListDefinition(listId).orderBy).toEqual([]);
+    // prettier-ignore
+    expect(getEvaluatedGrid(model, "A1:E5")).toEqual([
+        ["Foo", "Bar",   "Date", "Probability", "Money!"],
+        [12,    "TRUE",  42474,  10,                74.4],
+        [1,     "TRUE",  42669,  11,                74.8],
+        [17,    "TRUE",  42719,  95,                   4],
+        [2,     "FALSE", 42715,  15,                1000],
+    ]);
+    const orderBy = [
+        {
+            name: "bar",
+            asc: false,
+        },
+        {
+            name: "pognon",
+            asc: true,
+        },
+    ];
```

Comment:
this is a bit too much. It can be shorten
```suggestion
    const orderBy = [
        { name: "bar", asc: false },
        { name: "pognon",  asc: true },
    ];
```